examples/kubernetes-deployment-example.yaml

```diff
@@ -0,0 +1,166 @@
+# This file contains a basic configuration for kubernetes
+
+# DEPLOYMENT
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: almond-server
+  # namespace <your namespace>
+spec:
+  selector:
+    matchLabels:
+      app: almond-server
+      role: backend
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: almond-server
+        role: backend
+    spec:
+      #hostNetwork: true
+      #dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: almond-server
+        image: stanfordoval/almond-server
+        env:
+        - name: TZ
+          value: Europe/London
+        - name: THINGENGINE_HOST_BASED_AUTHENTICATION
+          value: local-ip
```

Comment:
Should this be `trusted-ip` and should you set `THINGENGINE_USE_REVERSE_PROXY`?

Reply:
Thanks for the feedback. 
I haven’t tested against the latest master; but I will and investigate your suggestions above.

I am running almond on an arm64 based platform so will need to recompile.

Reply:
I have now build an arm64 image from the latest master branch and tagged it 1.99.0.
However, could you point me in the direction of where, for example; THINGENGINE_HOST_BASED_AUTHENTICATION and THINGENGINE_HAS_REVERSE_PROXY are defined so that I can carry out some further tests.

Reply:
They are defined/used in config.js